.circleci/config.yml

```diff
@@ -26,7 +26,7 @@ jobs:
               command: |
                   export PATH=~/miniconda/bin:$PATH
                   conda update --yes --quiet conda
-                  conda create -n testenv --yes pip python=3.6
+                  conda create -n testenv --yes pip python=3.7
```

Comment:
Are you certain this has nothing to do with this PR? One way to verify is to make a "blank PR" with a tiny doc change and see if you have the same CircleCI problem. If so, I can live with this. It's not good to create such tight bleeding edge Python dependency. Because Neuron doesn't work on 3.8, it means we have only one version of Python for which MPI backend would work

Reply:
Switching back to 3.6, works on master (#272)